### ISSUE TITLE
Voice conversion effect?

### ISSUE THREAD

**Issue by nosudden** [kyungdeuk/NVCNet-pytorch#1]
Thanks to the open source NVC-Net pytorch implementation. I use this network structure + additional training code, the loss and training parameters are consistent with the reference paper, and the voice conversion effect is very poor, how is your training effect and is it convenient to share the training code?

Looking forward to your reply!
